mmv1/templates/terraform/examples/vmware_engine_private_cloud_full.tf.tmpl

```diff
@@ -14,6 +14,28 @@ resource "google_vmwareengine_private_cloud" "{{$.PrimaryResourceId}}" {
       node_count   = 1
       custom_core_count = 32
     }
+    autoscaling_settings {
+      autoscaling_policies {
+        autoscale_policy_id = "autoscaling-policy"
+        node_type_id = "standard-72"
+        scale_out_size = 1
+        cpu_thresholds {
+          scale_out = 80
+          scale_in  = 15
+        }
+        consumed_memory_thresholds {
+          scale_out = 75
+          scale_in  = 20
+        }
+        storage_thresholds {
+          scale_out = 80
+          scale_in  = 20
+        }
+      }
+      min_cluster_node_count = 3
+      max_cluster_node_count = 8
+      cool_down_period = "1800s"
```

Comment:
Can you please confirm that if these three fields will be returned in the response when they are not in the request? Thanks.